packages/xml-odata-annotation-converter/src/parser/metadata.ts

```diff
@@ -260,6 +260,29 @@ function getTypeForNavigationProperty(context: Context, element: XMLElement): st
     return undefined;
 }
 
+/**
+ * Get OData target kinds for a metadata element.
+ *
+ * @param metadata element.
+ * @returns OData target kinds.
+ */
+function getEdmTargetKinds(element: MetadataElementProperties): TargetKind[] {
```

Comment:
Since we only use `element.kind` here why not we use that as the parameter instead of the whole element? Then we can also use it easier when creating metadata elements, because we don't need the element itself, just its kind.

Reply:
ok

Reply:
done